extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -68,17 +79,28 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
       true /** callToInitialize */
     );
 
-    this.storeService_.subscribe(StateProperty.RTL_STATE, (rtlState) => {
-      this.onRtlStateUpdate_(rtlState);
-    });
+    this.storeService_.subscribe(StateProperty.RTL_STATE, (rtlState) =>
+      this.onRtlStateUpdate_(rtlState)
+    );
 
     this.storeService_.subscribe(
       StateProperty.PAGE_SIZE,
-      (pageSizeState) => {
-        this.flipTagIfOffscreen_(pageSizeState);
-      },
+      (pageSizeState) => this.flipTagIfOffscreen_(pageSizeState),
       true /** callToInitialize */
     );
+
+    this.storeService_.subscribe(StateProperty.CURRENT_PAGE_ID, (id) =>
+      this.toggleShoppingTagActive_(id)
+    );
+  }
+
+  /**
+   * @param {!string} id
```

Comment:
Primitives such as string/number/boolean are always non-nullable by default, so you can remove the redundant `!`